src/js/binaryen.js-post.js

```diff
@@ -3129,10 +3122,7 @@ Object.defineProperty(Module, 'ready', {
       } else if (runtimeInitialized) {
         resolve(Module);
       } else {
-        pendingPromises.push({
-          resolve: resolve,
-          reject: reject
-        });
+        pendingPromises.push({ resolve, reject });
```

Comment:
I remember that modern syntax like this can be a problem for Closure Compiler with Emscripten's default settings disallowing some ES5+ features. But CI appears to pass here, so not sure if that's relevant?

Reply:
It should compile this [without any problems](https://closure-compiler.appspot.com/home#code%3D%252F%252F%2520%253D%253DClosureCompiler%253D%253D%250A%252F%252F%2520%2540compilation_level%2520SIMPLE_OPTIMIZATIONS%250A%252F%252F%2520%2540output_file_name%2520default.js%250A%252F%252F%2520%253D%253D%252FClosureCompiler%253D%253D%250A%250Afunction%2520test(resolve%252C%2520reject)%2520%257B%250A%2520%2520return%2520%257B%2520resolve%252C%2520reject%2520%257D%253B%250A%257D%250A%250Awindow%255B'test'%255D%2520%253D%2520test%253B%250A%250A)

Reply:
The options or compiler version in your snippet might be different than those used by Emscripten. Mentioning because features like `let` and arrow functions make Emscripten explode. Seemed like even though Closure Compiler recognizes the syntax, Emscripten doesn't enable it.

Reply:
Alright reverted this just in case.